dataeng/resources/dbt-run.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -ex
+
+# Setup to run dbt commands
+cd $WORKSPACE/warehouse-transforms
+
+# Using --first-parent flag helps to avoids any intermediate commits that developers sometimes leave and merge without squashing. It helps us ignoring the noise in commit history.
+HEAD_COMMIT=$(git log --first-parent origin/master --format='%H' --max-count=1)
+
+if git log --format=%B -n 1 $HEAD_COMMIT | grep 'Schema Builder automated dbt update at' -q; then isSchemaBuilderPR="true"; else isSchemaBuilderPR="false"; fi
```

Comment:
nit: can you split this onto multiple lines for readability?

Reply:
yes, done that